pathz/pathz.proto

```diff
@@ -72,60 +67,32 @@ service Pathz {
   //     in the new policy and validates that the new policy "works".
   //     Additionally the client should access an OpenConfig path that is
   //     not allowed and the attempt must fail proving that the OpenConfig
-  //     Path-based Authz Policy "works".
-  //     Once verfied, the client then proceeds to finalize the rotation.
-  //     If the new verification did not succeed the client will cancel the
+  //     gNMI Path-based Authorization Policy "works".
+  //     Once verified, the client then proceeds to finalize the rotation.
+  //     If the verification did not succeed the client will cancel the
   //     RPC thereby forcing the target to perform a rollback of the new
-  //     OpenConfig Path-based Authz Policy.
+  //     OpenConfig gNMI Path-based Authorization Policy.
   //
   //   Step 4: Final commit.
   //     Client ---> FinalizeRequest ----> Target
```

Comment:
there isn't  a defined finalizeResponse, what happens if there's an error finalizing?
Is there a reason to not have FinalizeResponse{ string error }  or similar sort of construction?